packages/scss/src/components/inputs/textfield/_textfield.material.scss

```diff
@@ -120,8 +120,30 @@
 	// STATES
 	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
-	.textfield-input {
+	// Inline validation
+	&.is-valid {
+		&::before {
+			content: "";
```

Comment:
Why removing the icon in material ?

Reply:
Yep, we talked about that with @Clotilde. The idea is to use only the bottom line to give succes / error informations.

Reply:
Then what is the need for the other mods? Aren't borders enough in these cases ?